runtime/integration-tests/Cargo.toml

```diff
@@ -9,7 +9,8 @@ repository = "https://github.com/centrifuge/centrifuge-chain"
 
 
 [dependencies]
-fudge = { git = "https://github.com/centrifuge/fudge", tag = "v0.0.7-polkadot-v0.9.17"}
+#fudge = { git = "https://github.com/centrifuge/fudge", tag = "v0.0.5-polkadot-v0.9.17"}
```

Comment:
```suggestion
```

Reply:
Wil adapt this once I release the latest fudge version.